src/cargo/util/config.rs

```diff
@@ -19,21 +18,19 @@ use self::ConfigValue as CV;
 pub struct Config<'a> {
     home_path: Path,
     shell: RefCell<&'a mut MultiShell>,
-    jobs: u32,
-    target: Option<string::String>,
-    rustc_version: string::String,
+    rustc_version: String,
     /// The current host and default target of rustc
-    rustc_host: string::String,
+    rustc_host: String,
+    values: RefCell<HashMap<String, ConfigValue>>,
+    values_loaded: Cell<bool>,
```

Comment:
Any reason this can't e folded into the above as `values: RefCell<Option<...>>`?